plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/cache/BaseCachingHiveMetastoreTest.java

```diff
@@ -163,14 +173,22 @@ public void tearDown()
 
     private ThriftMetastore createThriftHiveMetastore()
     {
-        return createThriftHiveMetastore(mockClient);
+        return createThriftHiveMetastore(mockClient, useBatchTableFetches);
     }
 
     private static ThriftMetastore createThriftHiveMetastore(ThriftMetastoreClient client)
     {
-        return testingThriftHiveMetastoreBuilder()
-                .metastoreClient(client)
-                .build();
+        return createThriftHiveMetastore(client, false);
+    }
+
+    private static ThriftMetastore createThriftHiveMetastore(ThriftMetastoreClient client, boolean useBatchTableFetches)
+    {
+        TestingThriftHiveMetastoreBuilder builder = testingThriftHiveMetastoreBuilder()
+                .metastoreClient(client);
+        if (useBatchTableFetches) {
+            builder.hiveConfig(new HiveConfig().setTranslateHiveViews(true));
+        }
```

Comment:
else set it to false. don't rely on the particular default value of the config.
otherwise, when config default changes, we lose test coverage withut anyone noticing

btw this is pretty indirect and deserves a code comment why "useBatchTableFetches = TranslateHiveViews"